### PR TITLE
JSONify Error Response

### PR DESCRIPTION
**Pull request by kdiluca**
To resolve https://github.com/valhalla/loki/issues/133

### CODE REVIEW

File: src/loki/service.cc [hunk 1]
```diff
@@ -250,26 +255,23 @@ namespace valhalla {
...
-            break;
+            return matrix(action->second, request_pt, info);
           case ISOCHRONE:
-            result = isochrones(request_pt, info);
-            break;
+            return isochrones(request_pt, info);
           case ATTRIBUTES:
-            result = attributes(request_pt, info);
-            break;
+            return attributes(request_pt, info);
```
Comment (kevinkreiser) [inline src/loki/service.cc]: returning from here will skip the long request processing below. i'm sure i probably said oh yeah look at that you can just return, so if i did, i was wrong and sorry for suggesting that!

File: src/loki/service.cc [hunk 2]
```diff
@@ -282,16 +284,10 @@ namespace valhalla {
...
-        return result;
       }
       catch(const std::exception& e) {
-        worker_t::result_t result{false};
-        http_response_t response(400, "Bad Request", e.what(), headers_t{CORS});
-        response.from_info(info);
-        result.messages.emplace_back(response.to_string());
         valhalla::midgard::logging::Log("400::" + std::string(e.what()), " [ANALYTICS] ");
-        return result;
+        return jsonify_error(400, "Bad Request", std::string(e.what()), info);
```
Comment (kevinkreiser) [inline src/loki/service.cc]: no need to convert `e.what()` to string here, it will happen implicitly

File: valhalla/loki/service.h [hunk 3]
```diff
@@ -13,11 +13,13 @@
 #include <valhalla/baldr/connectivity_map.h>
 #include <valhalla/sif/costfactory.h>
 
+using namespace prime_server;
```
Comment (kevinkreiser) [inline valhalla/loki/service.h]: we shouldnt put `using namespace` in headers beacuse then anything that includes them includes the whole namespace. its fine to put this in the source file though.

**Comment by kevinkreiser** [comment]
i love the simplification, this looks great! other than the comments above :rocket: !

File: src/loki/service.cc (hunk 1 above)
Comment (kdiluca) [inline src/loki/service.cc]: No, you didn't, I was just trying to simplify and overlooked the long request.  Thanks for catching.

**Pull request merged**